Description: Increase test timeout to fix FTBFS on some architectures

--- haskell-skylighting-core-0.12.3.1.orig/test/test-skylighting.hs
+++ haskell-skylighting-core-0.12.3.1/test/test-skylighting.hs
@@ -224,7 +224,7 @@ p_no_drop cfg syntax t =
 
 noDropTest :: TokenizerConfig -> [Text] -> Syntax -> TestTree
 noDropTest cfg inps syntax =
-  localOption (mkTimeout 25000000)
+  localOption (mkTimeout 250000000)
   $ testCase (Text.unpack (sName syntax))
   $ mapM_ go inps
     where go inp =
